deps/wazuh_testing/wazuh_testing/tools/system.py

```diff
@@ -60,8 +144,21 @@ def move_file(self, host: str, src_path: str, dest_path: str = '/var/ossec/etc/o
         dest_path (str): Destination path
         check (bool, optional): Ansible check mode("Dry Run"), by default it is enabled so no changes will be applied.
         """
-        self.get_host(host).ansible("copy", f"src={src_path} dest={dest_path} owner=wazuh group=wazuh mode=0775",
-                                    check=check)
+        system = 'linux'
+        result = None
+
+        if 'os_name' in self.get_host_variables(host):
+            host_os_name = self.get_host_variables(host)['os_name']
+            if host_os_name == 'windows':
+                system = 'windows'
+
+        if system == 'linux':
+            result = self.get_host(host).ansible("copy", f"src={src_path} dest={dest_path} owner=wazuh group=wazuh mode=0644",
+                                        check=check)
+        else:
+            result = self.get_host(host).ansible("ansible.windows.win_copy", f"src='{src_path}' dest='{dest_path}'", check=check)
+
+        return result
```

Comment:
```suggestion
        if self.get_host_variables(host)['os_name'] == 'windows':
            self.get_host(host).ansible("ansible.windows.win_copy", f"src='{src_path}' dest='{dest_path}'", check=check)
        else:
            self.get_host(host).ansible("copy", f"src={src_path} dest={dest_path} owner=wazuh group=wazuh mode=0644",
                                        check=check)
```

Why are we changing the file permissions?